Gruntfile.js

```diff
@@ -49,7 +53,7 @@ module.exports = (grunt) => {
               loader: 'babel-loader',
               options: {
                 presets: [preset],
-                plugins: ['@babel/plugin-transform-object-assign'],
+                plugins: ['@babel/plugin-transform-object-assign', '@babel/plugin-proposal-class-properties'],
```

Comment:
Not sure if the project is using a formatter on config files, but I'd suggest each list item on a new line.